style.css

```diff
@@ -0,0 +1,386 @@
+* {
+	box-sizing: border-box;
+	margin: 0;
+	font-size: 10px;
```

Comment:
font size 10px should be set on the `html` element (see rem in our notes)

Reply:
done